srcpkgs/busybox/template

```diff
@@ -50,37 +71,41 @@ do_configure() {
 }
 
 do_build() {
-	make -C busybox SKIP_STRIP=y ${makejobs}
-	make -C busybox-static SKIP_STRIP=y ${makejobs}
+	for t in busybox-core busybox busybox-static busybox-huge; do
+		make -C "${t}" SKIP_STRIP=y "${makejobs}"
+	done
 }
 
 do_check() {
-	# Copy the testsuite into the busybox and it's static counterpart
-	# directory
-	cp -r src/testsuite busybox
-	cp -r src/testsuite busybox-static
-
-	# This is required by the testsuite
-	cp src/scripts/echo.c busybox/scripts
-	cp src/scripts/echo.c busybox-static/scripts
-
-	# Run testsuite for busybox
-	(
-	cd busybox/testsuite
-	SKIP_KNOWN_BUGS=yes ./runtest -v
-	)
-
-	# Run testsuite for busybox-static
-	(
-	cd busybox-static/testsuite
-	SKIP_KNOWN_BUGS=yes ./runtest -v
-	)
+	for t in busybox-core busybox busybox-static busybox-huge; do
+		# Copy the testsuite over
+		cp -r src/testsuite "${t}"
+		# Required by the testsuite
+		cp src/scripts/echo.c "${t}/scripts"
+		# Run the testsuite
+		(
+		cd "${t}"
+		SKIP_KNOWN_BUGS=yes ./runtest -v
+		)
+	done
 }
 
 do_install() {
 	vbin busybox/busybox_unstripped busybox
 	vman busybox/docs/busybox.1
 	vsv busybox-ntpd
+	vsv busybox-syslogd
+	vsv busybox-klogd
+}
+
+busybox-core_package() {
+	short_desc+=" (essential applets)"
+	conflicts="busybox>=0"
```

Comment:
conflicts ?

Reply:
Yeah all the paths conflict and it doesn't make sense to use more than one of them. It is a pick one type of deal.

Reply:
ah yes i derped out, i forgot busybox is a single binary